modules/osdk-java-generate-crd.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/java/osdk-java-tutorial.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-java-generate-crd_{context}"]
+= Generating CRD manifests
+
+After the API is defined with `MemcachedSpec` and `MemcachedStatus` files, you can generate CRD manifests.
+
+.Procedure
+
+* Run the following command from the previously created `memcached-operator` directory to generate the CRD:
++
+[source,terminal]
+----
+$ mvn clean install
+----
+
+.Verification
+
+* The CRD is stored in the `/target/kubernetes` directory. The contents of the `memcacheds.cache.example.com-v1.yml` file is shown below:
```

Comment:
```suggestion
* Verify the contents of the CRD in the `target/kubernetes/memcacheds.cache.example.com-v1.yml` file as shown in the following example:
```
Suggestion to remove passive voice and to avoid [_below_](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b).

Reply:
+1 to this suggestion